### ISSUE TITLE
Image similarity threshold configuration?

### ISSUE THREAD

**Issue by Thiece** [lene/DuplicateImages#2]
I can't find a description of its default case similarity threshold option, and there is no way to change this threshold

**Comment by lene** [comment]
True, that is not supported. Having a threshold is not how it works with image hashes, instead you have to change the number of bits the image hash uses. I will add an option for this soon(-ish, after my vacation).
